quartz/components/Comments.tsx

```diff
@@ -5,6 +5,7 @@ import script from "./scripts/comments.inline"
 
 type Options = {
   provider: "giscus"
+  respectFrontmatter?: boolean
```

Comment:
actually we can just have it check frontmatter by default instead of gating over an option.

I don't really see a reason when this is not useful.

Reply:
I didn't want to change the default behavior, that's why I added the option. What would you suggest instead? Keep comments enabled by default on every page unless there is a `comments: false` frontmatter or disable comments by default & only display them if `comments: true`?

Reply:
I'd assuming default true, as the comments component is not enabled by default. Adding it to the layout is opt-in already.

Reply:
if users set `Plugin.Comments()` the would expect to include all comments by default.

Then we can check for frontmatter, like what saber suggest here

Reply:
Right makes sense, I saw it from my perspective where I only wanted comments enabled on a specific set of Pages, where setting false on a majority would have been more effort, but since it's a component & not a config option, I agree. Will commit in a second